public/app/percona/shared/services/user/User.service.ts

```diff
@@ -0,0 +1,17 @@
+import { CancelToken } from 'axios';
+import { api } from 'app/percona/shared/helpers/api';
+import { UserConnectedResponse } from './User.types';
+
+const BASE_URL = '/v1/Platform';
+
+export const UserService = {
```

Comment:
I also see that the guys usually cover the work of hooks in .service.ts files with unit tests, so it seems to me that this should also be covered by tests, maybe I'm wrong...

Reply:
@solovevayaroslavna This service only does an API call, we don't have a hook here or any business logic, if I got this correctly.

Reply:
I agree there is no need for unit test here. Resolving.